app/views/catalog/_print.html.erb

```diff
@@ -0,0 +1,23 @@
+<%# file _print.html.erb %>
+<% @documents.entry.each do |document| %>
+
+  <h3 class="modal-title"><%= document_heading(document) %></h3>
+
+  <%# PUT document.send(:print) here; example from citation below %>
+
+       <% if document.respond_to?(:export_as_mla_citation_txt) %>
+         <h4><%#= t('blacklight.citation.mla') %></h4>
+         <%3= document.send(:export_as_mla_citation_txt).html_safe %><br/><br/>
+       <% end %>
+
+       <% if document.respond_to?(:export_as_apa_citation_txt) %>
+         <h4><%#= t('blacklight.citation.apa') %></h4>
+         <%#= document.send(:export_as_apa_citation_txt).html_safe %><br/><br/>
```

Comment:
Also, in the interest of at least pretending we're doing some code review: this code seems overly defensive to me. The list of "responds to" and then using `#send(:methodname)` feels like it's designed for a loop (which we're not using) and assumes that the representable objects don't have a unified interface (which they darn well should). Let's think about a cleaner way to do this once we get this damn thing out the door.

Reply:
Okay. I'm following the citation's blacklight pattern here, which may be overly defensive, especially for a print function.

Reply:
And I removed the "responds to" ifs.